singlem/condense.py

```diff
@@ -294,6 +314,9 @@ def _condense_domain_to_species(self, sample, sample_otus, markers, target_domai
                 node_list = node_list_queue.get()
 
                 # Calculate stat for this set of markers
+                if avg_num_genes_per_species is not None: # Running in viral mode
+                    m_coverages = [m.get_full_coverage() for m in node_list]
+                    total_num_markers = max(avg_num_genes_per_species, len(m_coverages))
```

Comment:
For species, where the number of markers is known, it should use that, otherwise the above